content/_components/shopify-admin-v2/index.md

```diff
@@ -44,16 +44,16 @@ To use this component you need to create a custom App:
 
 Component credentials configuration fields:
 
-* **Store name** (string, required) - Provide you store name here, can be found in [Admin panel](https://accounts.shopify.com/store-login) as a part of store url
+* **Store name** (string, required) - Provide you store name here, can be found in [Admin panel](https://accounts.shopify.com/store-login) as a part of store URL.
 
 ![Store Name](img/store-name.png)
 
 * **Admin API access token** (string, required) - this token you will get after app creation (look at instructions above)
-* **API version** (string, required) - Provide API version you are going to work with. Currently component tested on `2023-01`, but should work with any available
+* **API version** (string, required) - Provide API version you are going to work with. Currently component tested on `2023-01`, but should work with any available.
```

Comment:
* **API version** (string, required) - Provide the API version you are going to work with. The component has been tested on `2023-01`, but should work with any available.